service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageOrdersSubmittedControllerTest.java

```diff
@@ -148,6 +155,9 @@
     @MockBean
     private DocmosisCoverDocumentsService documentService;
 
+    @MockBean
+    private ManageOrdersEventBuilder manageOrdersEventBuilder;
```

Comment:
Shouldn't be mocking this in the integration test. These sorts of tests are supposed to use the real components as much as possible, the only reason the others are mocked is because they connect to external services.